### PR TITLE
Sepolia TheMerge configs

### PR DESCRIPTION
**Pull request by marcindsobczak**
## Changes:
- add merge configs for sepolia

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [x] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner/configs/sepolia.cfg [hunk 1]
```diff
@@ -20,5 +20,30 @@
     "PivotNumber": 1301000,
     "PivotHash": "0x0f66dfd68f63261eb91253eacb7df90192c751f85204f60840ddb12ccd91227b",
     "PivotTotalDifficulty": "6921232492055148"
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "EnabledModules": [
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/configs/sepolia.cfg:29]: this is same as DefaultModules, we can drop it?

File: src/Nethermind/Nethermind.Runner/configs/sepolia.cfg [hunk 2]
```diff
@@ -20,5 +20,30 @@
...
+    "Parity",
+    "Health"
+    ],
+    "AdditionalRpcUrls": [
+      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
+    ]
+  },
+  "Merge": {
+    "Enabled": true,
+    "TerminalTotalDifficulty": "100000000000000000000000"
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/configs/sepolia.cfg:47]: placeholder?

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/configs/sepolia.cfg (hunk 2 above)
Comment (marcindsobczak) [reply src/Nethermind/Nethermind.Runner/configs/sepolia.cfg:35]: Yes

File: src/Nethermind/Nethermind.Runner/configs/sepolia.cfg (hunk 1 above)
Comment (marcindsobczak) [reply src/Nethermind/Nethermind.Runner/configs/sepolia.cfg]: Actually yes, removed

**Pull request merged**